go.mod

```diff
@@ -28,11 +28,11 @@ require (
 	github.com/hashicorp/go-version v1.0.0
 	github.com/hashicorp/hcl v0.0.0-20170504190234-a4b07c25de5f // indirect
 	github.com/hashicorp/hcl2 v0.0.0-20171003232734-44bad6dbf549 // indirect
-	github.com/hashicorp/hil v0.0.0-20170512213305-fac2259da677 // indirect
+	github.com/hashicorp/hil v0.0.0-20190212132231-97b3a9cdfa93 // indirect
```

Comment:
I'd recommend removing any `go.mod` and `go.sum` changes to this pull request as they should be unrelated to this change and likely cause merge conflicts (as they are currently).